source/common/res/features/budget-quick-switch/main.js

```diff
@@ -0,0 +1,42 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
+    ynabToolKit.activityTransactionLink = (function () {
+      let applicationController = ynabToolKit.shared.containerLookup('controller:application');
+      let userBudgetsController = ynabToolKit.shared.containerLookup('controller:users/budgets');
+
+      function populateBudgetList() {
+        let budgets = userBudgetsController.get('budgets');
+        let currentBudgetId = applicationController.get('activeBudgetVersion').get('entityId');
+        let $openBudgetListItem = $('.modal-select-budget').find('.modal-select-budget-open').parent();
+
+        budgets.forEach((budget) => {
+          if (budget.get('budgetVersionId') === currentBudgetId) return;
+
+          let budgetListItem = $(`<li>
+                                    <button>
+                                      <i class="flaticon stroke mail-1"></i>
+                                      ${budget.get('budgetVersionName')}
```

Comment:
you're probably right...the template literals are just going to get converted to string concatenation by babel anyway...I'll convert this when I get a chance :D